src/current/v23.2/create-virtual-cluster.md

```diff
@@ -0,0 +1,110 @@
+---
+title: CREATE VIRTUAL CLUSTER
+summary: The CREATE VIRTUAL CLUSTER statement creates a new target virtual cluster for physical replication.
+toc: true
+docs_area: reference.sql
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include enterprise-feature.md %}
+
+{% include_cached new-in.html version="v23.2" %} The `CREATE VIRTUAL CLUSTER` statement starts a [physical cluster replication job]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}). `CREATE VIRTUAL CLUSTER` only supports starting a replication stream as part of the [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) workflow.
+
+{% include {{ page.version.version }}/physical-replication/phys-rep-sql-pages.md %}
+
+## Required privileges
+
+`CREATE VIRTUAL CLUSTER` requires one of the following privileges:
+
+- The `admin` role.
+- The `MANAGEVIRTUALCLUSTER` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges) allows the user to run all the related `VIRTUAL CLUSTER` SQL statements for physical cluster replication.
+- The `REPLICATION` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges) allows a user to connect to stream data out of a virtual cluster for physical cluster replication.
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM MANAGEVIRTUALCLUSTER TO user;
+~~~
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/physical-replication/create-virtual-cluster-diagram.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+------------
+`virtual_cluster_name` | The name for the new virtual cluster.
+`LIKE virtual_cluster_spec` | Ensures the virtual cluster on the standby is created with the correct [capabilities](#capabilities).
```

Comment:
The description here should be: "Create a virtual cluster with the same capabilities and settings as another tenant.